ctapipe/image/muon/tests/test_features.py

```diff
@@ -14,6 +19,48 @@ def test_ring_containment():
 
     assert(ring_cont <= 1. and ring_cont >= 0.)
 
+
+def test_ring_completeness():
+
+    angle_ring = np.linspace(0, 2 * mt.pi, 360.)
+    x = np.cos(angle_ring) * u.m
+    y = np.sin(angle_ring) * u.m
+    pe = np.random.uniform(0, 100, len(x))
+    ring_radius = 1. * u.m
+
+    ring_center_x = 0. * u.m
+    ring_center_y = 0. * u.m
+
+    ring_comp = ring_completeness(
+        x, y, pe, ring_radius,
+        ring_center_x, ring_center_y,
+        30, 30)
```

Comment:
Sometimes functions, which need more than 3 or 4 parameters (apart from plotting functions, which are known to take a virtually unlimited amount of parameters :-D) are a hint, that there is a little class waiting to be implemented.

I could imagine a little `Ring` class (just a good old C-like-struct in python maybe a `collections.namedtuple`) might help here.